notebooks/tutorial/8 - Getting tomes from the ADX.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "75814a9a",
+   "metadata": {},
+   "source": [
+    "------------\n",
+    "\n",
+    "**FPS Critic Inc., owner of PureSkill.gg, is not liable for any AWS\n",
+    "costs you incur. Only run this notebook if you understand and accept\n",
+    "the AWS billing implications.**\n",
+    "\n",
+    "------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "288a0939",
+   "metadata": {},
+   "source": [
+    "# Getting pre-built tomes from the ADX\n",
+    "\n",
+    "The csds files are nice to work with because they contain full player telemetry and game event data. However, these are quite large and for many purposes\n",
```

Comment:
```suggestion
    "The csds files are powerful to work with because they contain full player telemetry and game event data. However, these are quite large and for many purposes. Tomes are often much smaller, so are inexpensive to download and share.\n",
```